### ISSUE TITLE
Where clause returning NULL results for Timesheets for some filters

### ISSUE THREAD

**Issue by phollida** [XeroAPI/Xero-NetStandard#232]
- Version 3.6.0

I'm trying to pull all our employees timesheet hours that occur within a calendar month using the payrollAUApi.GetTimesheetsAsync functions with a where clause to filter the start and end dates.

Take September for example,  This where clause works fine to pull all timesheets that start and end in the month:
```
payrollAUApi.GetTimesheetsAsync(accessToken, tenantID, where:"StartDate >= DateTime(2020,09,01) and EndDate <= DateTime(2020,09,30)"
```
But this doesn't return timesheets that begin and end outside the calendar month.

Anything more complex returns null Timesheets.  For example these filters all return null through the NetStandard wrapper, but they all return correct results through the Delevoper API Previewer using exactly the same syntax:

```
payrollAUApi.GetTimesheetsAsync(accessToken, tenantID, where:"StartDate >= DateTime(2020,09,01) and StartDate <= DateTime(2020,09,30)"
```
or the one I really need to do:
```
payrollAUApi.GetTimesheetsAsync(accessToken, tenantID, where:"(StartDate >= DateTime(2020,09,01) and StartDate <= DateTime(2020,09,30)) or (EndDate >= DateTime(2020,09,01) and EndDate <= DateTime(2020,09,30))"

```
both return null through the wrapper, but work fine in the API Previewer.

**Comment by jeffvictorino** [comment]
I have encountered the same issue with payrollAuApi.GetTimesheetsAsync() even without any where clause. 

When I saw this issue, it gave me a clue to apply a where clause and I have found out that on that specific tenant, if I pull any Timesheet from 2017 onwards, it return null, where: "(StartDate >= DateTime(2018, 01, 01))" worked and I was able to fetch timesheets. It looks like it depends on the timesheet data and based on it serialisation fails on the NetStandard client.
